src/logic/queryParams.js

```diff
@@ -17,7 +17,8 @@ import {
  *   in a generic URL query parameter and also in localStorage.
  *
  * There are two mapping functions:
- * - `mapSearchQuery` which works with `SearchQuery` model. This model is mostly used on SearchPage only.
+ * - `mapSearchQuery` which works with `SearchQuery` model. This model is mostly used on IssuePage only,
+ *   as normally you may want to set additionalParameters (e.g. reset the page number).
```

Comment:
This statement does not seem to be correct because `additionalParamters` are supported by both mappers.

Reply:
currently `mapSearchQuery` doesn't support additional parameters:
```javascript
export const mapSearchQuery = () => ({
  ...searchQueryGetter(),
  ...searchQuerySetter(),
});
```

Reply:
So it's not even correct but rather contradictory :)

Reply:
uh? why?

Reply:
IMO, the most important thing to document here is the possibility of having the getter and the setter for the search query (or for th filters - even if I would have avoided  the mapFilters); the mapSearchQuery is almost never used.